activities.el

```diff
@@ -881,13 +889,18 @@ with prefix argument, choose another activity."
 
 (defun activities-bookmark-store (activity)
   "Store a `bookmark' record for ACTIVITY."
-  (bookmark-maybe-load-default-file)
```

Comment:
Not that I distrust what you said about this, but I feel like I added this because I found it necessary in another package of mine, so I'm not sure it should be removed here.  I wouldn't want to introduce a bug.